StackUsingLinkedList.java

```diff
@@ -0,0 +1,55 @@
+import java.io.*;
+public class StackUsingLinkedList{
+       public static class Node{
+		          int data;
+		          Node next;
+		   
+		         Node (int d){
+			          data=d;
+			          next=null;
+		          }
+	   }
+	  public static Node top;
+	  
+    public static void push(int x){
+		         Node newNode=new Node(x);   
+		         if(top==null){
+				        newNode.next=null;
+           
+			        }	
+			      else {
+				      newNode.next=top;
+			      }
+			top=newNode;
+	   }
+	   public static void pop(){
+		        if(top==null){
+			            System.out.println("The Stack is empty\n");
+		         }
+		        else{
+			            top=top.next;
+		        }
+	    }
+		public static void print(Node top) {
+			    Node temp=top;
+			    while(temp!=null){
+				        System.out.print(temp.data+" ");
+				        temp=temp.next;
+			    }
+			    System.out.print("\n");
+		}
+	public static void main(String[] args) {
```

Comment:
@atulya0622  You didn't use the linkedlist u created last week at all,the code would take max 20 lines if u use that...DRY is an important coding practice...DONT REPEAT YOURSELF..reuse the code